src/pages/_app.tsx

```diff
@@ -6,8 +6,10 @@ import { AppPropsWithLayout } from "@/lib/types"
 
 import ThemeProvider from "@/components/ThemeProvider"
 
+import "@docsearch/css"
 import "@/styles/global.css"
 import "@/styles/fonts.css"
+import "@/styles/docsearch.css"
```

Comment:
Changed the `eslint` import sort config to allow this. The order of these imports is important to override the external styles with our custom styles.